packages/tools/generator-component/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/generator-component",
   "description": "Generator Component – A generator for Fozzie components",
-  "version": "2.6.0",
+  "version": "2.8.0",
```

Comment:
Why 2.8.0 and not 2.7.0?

Reply:
There was a 2.7.0 before, but I forgot to actually bump the version number last time, so I just skipped the version